### DISCUSSION TITLE
lombok plugin does not work

### DISCUSSION THREAD

**Discussion by zhuanshenlikai** [eclipse-theia/theia#12382] [Q&A] [answered]
I use lombok in the java project.I download the plugin of lombok(gabrielbb.vscode-lombok),however it does not seem to work.
I touch the button of DEBUG,the project is start to compile,but get the below error:

![image](https://user-images.githubusercontent.com/12570095/230261441-0fd1db1a-34ee-4490-8202-695d3fc68825.png)

It seems the GET/SET method not generated.The theia version is 1.32.0. I do not know if my operation has any mistakes.I am new at this.Can anyone help me resolve it?thanks.

**Comment by msujew** [comment, marked as answer]
Hey @zhuanshenlikai,

Theia only provides the API for consuming vscode extensions, but we usually can offer only very little help on extension specific issues. They are usually developed by people that are completely unrelated to the Theia project. You might want to ask the extension developers over [here](https://github.com/microsoft/vscode-lombok) directly.

**Reply by zhuanshenlikai** [reply]
ok,thank you！
